README.md

```diff
@@ -99,6 +99,49 @@ If you are interested in running on different hardware or with different models,
 
 &nbsp;
 
+### Optimization flags
+
+The following table demonstrates the impact of sequentially applying various optimization techniques available on torchtune to the Llama 3B model. Each technique is added on top of the previous one, except for LoRA and QLoRA, which do not use `opt_in_bwd` or `Adamw8bit`.
+
+**Baseline:**
+- **Model:** Llama 3B
+- **Batch size:** 5
```

Comment:
I suspect I know why you did this but imo it's a bit strange to use 5 as a batch size

Reply:
Also should clarify that the baseline is full finetune